apps/laboratory/src/utils/SiweUtils.ts

```diff
@@ -65,9 +65,18 @@ export const siweConfig = createSIWEConfig({
       await signOut({
         redirect: false
       })
+      // Some times the first signOut doesn't delete the cookie, so we need to sign out again
+      const session = await getSession()
+      if (session) {
+        await signOut({
+          redirect: false
+        })
+      }
 
       return true
     } catch (error) {
+      console.warn('@@siweConfig: signOut error', error)
```

Comment:
Nit just for consistency
```suggestion
      console.warn('SiweUtils - SignOut error', error)
```